service/controller/v15/key/key.go

```diff
@@ -108,10 +112,13 @@ func BucketName(customObject v1alpha1.AWSConfig, accountID string) string {
 	return fmt.Sprintf("%s-g8s-%s", accountID, ClusterID(customObject))
 }
 
-// TODO what should the path look like when we want to use the version bundle
-// version of the operator?
-func BucketObjectName(templateVersion string, prefix string) string {
-	return fmt.Sprintf("cloudconfig/%s/%s", templateVersion, prefix)
+// BucketObjectName computes the S3 object path to the actual cloud config.
+//
+//     /version/3.4.0/cloudconfig/v_3_2_5/master
+//     /version/3.4.0/cloudconfig/v_3_2_5/worker
```

Comment:
Thinking more about that, when we have release-operator we could make those paths prettier like `/version/v3.4.0/cloudconfig/v3.2.5/master`